go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/convox/convox
 
-go 1.19
+go 1.21
```

Comment:
can you update this also go 1.23, since i see you updated it to 1.23 in every other places